tuplex/codegen/src/StandardModules.cc

```diff
@@ -56,6 +56,40 @@ namespace tuplex {
                     m->addAttribute(make_shared<Symbol>(name, name, python::Type::makeFunctionType(python::Type::propagateToTupleType(type), python::Type::F64), SymbolType::FUNCTION));
                 }
             }
+            
+            // math.isnan
+            auto isnanSym = make_shared<Symbol>("isnan", "isnan", python::Type::makeFunctionType(python::Type::propagateToTupleType(python::Type::F64), python::Type::BOOLEAN), SymbolType::FUNCTION);
+            isnanSym->addTypeIfNotExists(python::Type::makeFunctionType(python::Type::propagateToTupleType(python::Type::I64), python::Type::BOOLEAN));
+            isnanSym->addTypeIfNotExists(python::Type::makeFunctionType(python::Type::propagateToTupleType(python::Type::BOOLEAN), python::Type::BOOLEAN));
+            m->addAttribute(isnanSym);
+
+            // math.isinf
+            auto isinfSym = make_shared<Symbol>("isinf", "isinf", python::Type::makeFunctionType(python::Type::propagateToTupleType(python::Type::F64), python::Type::BOOLEAN), SymbolType::FUNCTION);
+            isinfSym->addTypeIfNotExists(python::Type::makeFunctionType(python::Type::propagateToTupleType(python::Type::I64), python::Type::BOOLEAN));
+            isinfSym->addTypeIfNotExists(python::Type::makeFunctionType(python::Type::propagateToTupleType(python::Type::BOOLEAN), python::Type::BOOLEAN));
+            m->addAttribute(isinfSym);
+
+            // math.isclose
+            // typing is:
+            // ({f64, i64, bool}, {f64, i64, bool}[, rel_tol={f64, i64, bool}, abs_tol={f64, i64, bool}]) -> bool
+            vector<python::Type> isclose_types{python::Type::BOOLEAN, python::Type::I64, python::Type::F64};
+            for(const auto& type : isclose_types) {
```

Comment:
maybe use a typer function for this? I think else 3^4 = 81 entries will get added to the table.